### PR TITLE
Add selected photos access sample

### PR DESCRIPTION
**Pull request by yrezgui**

### CODE REVIEW

File: samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+                                onClick = {
+                                    requestPermissions.launch(
+                                        arrayOf(READ_MEDIA_IMAGES, READ_MEDIA_VIDEO),
+                                    )
+                                },
+                            ) {
+                                Text("Both")
+                            }
+                        }
+                    } else if (Build.VERSION.SDK_INT == Build.VERSION_CODES.TIRAMISU) {
```
Comment (marcelpinto) [inline samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt:128]: I read this like in T you should request both permissions and in U you should split them.

I think you should show the split from T onwards but in U add the READ_MEDIA_VISUAL_USER_SELECTED in the request.

File: samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+                        TextButton(
+                            onClick = {
+                                requestPermissions.launch(
+                                    arrayOf(READ_MEDIA_IMAGES, READ_MEDIA_VIDEO),
+                                )
+                            },
+                        ) {
+                            Text("Request full library access")
```
Comment (marcelpinto) [inline samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt:136]: Also, this is not full library access like the RES.

**Review by marcelpinto — CHANGES_REQUESTED** [review]
LGTM with minor comments

**Comment by yrezgui** [comment]
Merging U & T logic makes sense but requesting `READ_MEDIA_VISUAL_USER_SELECTED` isn't useful here

File: samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt (hunk 1 above)
Comment (yrezgui) [reply samples/storage/src/main/java/com/example/platform/storage/mediastore/SelectedPhotosAccess.kt]: Merging U & T logic makes sense but requesting `READ_MEDIA_VISUAL_USER_SELECTED` isn't useful here

**Pull request merged**